evm_arithmetization/src/fixed_recursive_verifier.rs

```diff
@@ -2075,16 +2103,36 @@ where
 
         for table in 0..NUM_TABLES {
             let (table_circuit, index_verifier_data) = &table_circuits[table];
+            if KECCAK_TABLES_INDICES.contains(&table) && !all_proof.use_keccak_tables {
+                root_inputs.set_target(
+                    self.root.index_verifier_data[table],
+                    F::from_canonical_u8(*index_verifier_data),
+                );
+                // generate and set a dummy `proof_with_pis`
+                let common_date = &table_circuit
+                    .shrinking_wrappers
+                    .last()
+                    .expect("")
```

Comment:
```suggestion
                    .expect("No shrinking circuits")
```
as in `prove_segment`